packages/field-plugin/src/messaging/pluginMessage/containerToPluginMessage/StateChangedMessage.ts

```diff
@@ -33,6 +34,8 @@ export const isStateMessage = (data: unknown): data is StateChangedMessage =>
   typeof data.language === 'string' &&
   hasKey(data, 'schema') &&
   isFieldPluginSchema(data.schema) &&
+  hasKey(data, 'userId') &&
```

Comment:
Thanks for this @ECJ222 🫂

Reply:
> I meant the warning messages:
[Vue warn]: Invalid prop: type check failed for prop "spaceId". Expected Number with value X, got String with value "X"

Oh, I don't think it's related to this PR.